### ISSUE TITLE
Terminology and descriptions in Ideogram are sometimes sloppy.

### ISSUE THREAD

**Issue by mkcor** [plotly/dash-bio#235]
Following interactions with @TahiriNadia, we have decided to replace the term 'organism' with 'species' (which is more fitting in the context).

There are also several typos in the docstrings, as spotted by @Bachibouzouk when implementing tests.

I (or someone else) will work on it once #201 is merged into `master`.
